cmd/veneur/main.go

```diff
@@ -120,11 +120,12 @@ func main() {
 	// Read forever!
 	for {
 		buf := packetPool.Get().([]byte)
-		n, _, err := serverConn.ReadFromUDP(buf)
+		n, addr, err := serverConn.ReadFromUDP(buf)
 		if err != nil {
 			log.WithError(err).Error("Error reading from UDP")
 			continue
 		}
+		veneur.Stats.Set("unique_sender_ips", addr.IP.String(), nil, 1.0)
```

Comment:
From my past tests, emitting a metric within the read-loop has a significant effect on throughput. We might should pass it to the parser so that the penalty is outside the tight loop here?